R/zchunk_LA117.tradbio.R

```diff
@@ -27,46 +26,35 @@ module_energy_LA117.tradbio_DISABLED <- function(command, ...) {
     all_data <- list(...)[[1]]
 
     # Load required inputs
-    A_regions <- get_data(all_data, "energy/A_regions")
-    iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
-    L1011.en_bal_EJ_R_Si_Fi_Yh <- get_data(all_data, "L1011.en_bal_EJ_R_Si_Fi_Yh")
+    L1011.en_bal_EJ_R_Si_Fi_Yh <- get_data(all_data, "temp-data-inject/L1011.en_bal_EJ_R_Si_Fi_Yh")
     A17.tradbio_curves <- get_data(all_data, "energy/A17.tradbio_curves")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+
+    # Calculating the max resource of tradbio as the maximum during the historical years
+    L117.maxSubResource_tradbio <- L1011.en_bal_EJ_R_Si_Fi_Yh %>%
+      filter(fuel == "biomass_tradbio", sector == "TPES") %>%
+      gather(Xyear, value, starts_with("X")) %>%
```

Comment:
Note this line will be removed, the next line will need to change, when we link up to the 'real' `L1011.en_bal_EJ_R_Si_Fi_Yh`.